app/routers/auth.py

```
@@ -51,6 +56,96 @@ async def kakao_callback(code: str = Query(...), db: AsyncSession = Depends(get_
+    }
+    google_auth_url = f"https://accounts.google.com/o/oauth2/auth?{urlencode(params)}"
+
+    return RedirectResponse(url=google_auth_url)
+
+
+@router.get("/oauth/google/callback")
+async def google_callback(
+    request: Request,
+    code: str,
+    state: str,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)